packages/rs-drive-abci/src/main.rs

```diff
@@ -74,17 +78,30 @@ pub fn main() {
                 config.core.rpc.password.clone(),
             )
             .unwrap();
-            drive_abci::abci::start(&config, core_rpc).unwrap()
+            drive_abci::abci::start(&config, core_rpc).unwrap();
+            Ok(())
         }
         Commands::Config {} => dump_config(&config),
+        Commands::Status {} => check_status(&config),
     }
 }
 
-fn dump_config(config: &PlatformConfig) {
+fn dump_config(config: &PlatformConfig) -> Result<(), String> {
     let serialized =
         serde_json::to_string_pretty(config).expect("failed to generate configuration");
 
     println!("{}", serialized);
+
+    Ok(())
+}
+
+/// Check status of ABCI server.
+///
+/// Returns 0
+/// TODO: For now, it's just a placeholder.
+fn check_status(_config: &PlatformConfig) -> Result<(), String> {
+    println!("OK");
```

Comment:
Will it difficult to send ping to abci port to make sure it's listening? I'm afraid always returning 0 will be confusing for monitoring systems